src/pint/event_toas.py

```diff
@@ -87,6 +81,17 @@ def create_mission_config():
         mission_config[mission]["fits_extension"] = ext
         mission_config[mission]["fits_columns"] = cols
 
+    # Allow local TOAs for those missions that have a load_<MISSION>_TOAs method
```

Comment:
Can you use getattr to check for the existence of a `f"load_{mission}_TOAs` method directly, and save maintaining this explicit list?

Reply:
That isn't trivial since the mission names are different than the load_XXX_TOAs names. Also, I'm not sure this whole thing is needed. The only purpose is to set `allow_local` and the only place that is used it to emit an error message that does not stop the code from running:
```
    if not mission_config[mission]["allow_local"] and timesys != "TDB":
        log.error("Raw spacecraft TOAs not yet supported for " + mission)
```
Maybe @matteobachetti has some future plans for `allow_local`?